index.html

```diff
@@ -861,6 +861,10 @@
         <section>
             <h3>Implicit Release of Pointer Capture</h3>
             <p>Immediately after firing the <code>pointerup</code> or <code>pointercancel</code> events, a user agent MUST clear the <a>pending pointer capture target override</a> for the <code>pointerId</code> of the <code>pointerup</code> or <code>pointercancel</code> event that was just dispatched, and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code> if necessary.</p>
+            <p>Immediately before firing <code>dragstart</code> event [[!HTML]],
+a user agent MUST clear the <a>pending pointer capture target override</a> for the pointer that caused the drag operation,
+and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code>
+before the <code>dragstart</code> event if necessary.</p>
```

Comment:
Nit: can remove "... before the `dragstart` event..." because the whole para is about an action before `dragstart`.